src/Documentation/Documentation.v4.0.shfbproj

```diff
@@ -38,7 +38,7 @@
     <HelpTitle>openstack.net API Reference Documentation</HelpTitle>
     <ContentPlacement>AboveNamespaces</ContentPlacement>
     <RootNamespaceTitle>API Reference</RootNamespaceTitle>
-    <WebsiteSdkLinkType>Msdn</WebsiteSdkLinkType>
+    <WebsiteSdkLinkType>None</WebsiteSdkLinkType>
```

Comment:
This change *might* break links back to the MSDN documentation. It should be easy to verify this by viewing the output for [`IObjectStorageProvider.CopyObject`](http://openstacknetsdk.org/docs/html/M_net_openstack_Core_Providers_IObjectStorageProvider_CopyObject.htm) and checking that the links to `System.String` still work in the **Parameters** section.